src/ios/AppsFlyerPlugin.m

```diff
@@ -35,81 +35,103 @@ - (void)initSdk:(CDVInvokedUrlCommand*)command
         [self.commandDelegate sendPluginResult:result callbackId:command.callbackId];
         return;
-    }
-    else{
-        if(isDeepLinking == YES){
+    } else {
+        if (isDeepLinking == YES) {
             [AppsFlyerLib shared].deepLinkDelegate = self;
-         }
+        }
 
-        [[AppsFlyerLib shared] setPluginInfoWith:AFSDKPluginCordova pluginVersion:@"6.12.2" additionalParams:nil];
+        // Initialize the SDK
+        [[AppsFlyerLib shared] setPluginInfoWith:AFSDKPluginCordova pluginVersion:@"6.13.0" additionalParams:nil];
         [AppsFlyerLib shared].appleAppID = appId;
         [AppsFlyerLib shared].appsFlyerDevKey = devKey;
         [AppsFlyerLib shared].isDebug = isDebug;
         [AppsFlyerLib shared].useUninstallSandbox = useUninstallSandbox;
+    }
 
 #ifndef AFSDK_NO_IDFA
-        //Here we set the time that the sdk will wait before he starts the launch. we take the time from the 'option' object in the app's index.js
-        if (@available(iOS 14, *)) {
-            if (waitForATTUserAuthorization != 0 && waitForATTUserAuthorization != nil){
-                [[AppsFlyerLib shared] waitForATTUserAuthorizationWithTimeoutInterval:waitForATTUserAuthorization.intValue];
-                   }
+    // Here we set the time that the SDK will wait before it starts the launch. We take the time from the 'option' object in the app's index.js
+    if (@available(iOS 14, *)) {
+        if (waitForATTUserAuthorization != 0 && waitForATTUserAuthorization != nil) {
+            [[AppsFlyerLib shared] waitForATTUserAuthorizationWithTimeoutInterval:waitForATTUserAuthorization.intValue];
         }
+    }
 #endif
-        [[AppsFlyerLib shared] start];
+    if (![initSdkOptions isKindOfClass:[NSNull class]]) {
+
+        id isConversionDataValue = nil;
+        isConversionDataValue = [initSdkOptions objectForKey:afConversionData];
+
+        if ([isConversionDataValue isKindOfClass:[NSNumber class]]) {
+            isConversionData = [(NSNumber*)isConversionDataValue boolValue];
+        }
+    }
 
-        //post notification for the deep link object that the bridge is set and he can handle deep link
+    if (shouldStartSdk == true) {
+        [self startSdk:command];
+    }
+
+    //post notification for the deep link object that the bridge is set and he can handle deep link
         [AppsFlyerAttribution shared].isBridgeReady = YES;
         [[NSNotificationCenter defaultCenter] postNotificationName:AF_BRIDGE_SET object:self];
         // Register for background-foreground transitions natively instead of doing this in JavaScript
         [[NSNotificationCenter defaultCenter] addObserver:self
                                                  selector:@selector(sendLaunch:)
                                                      name:UIApplicationDidBecomeActiveNotification
                                                    object:nil];
```

Comment:
Did you check the scenario where you set `shouldStartSdk = NO`, send the app to background and then to foreground again? according to this it will trigger the `sendLaunch` method and start the SDK

Reply:
fixed